R/utilities-plots.R

```diff
@@ -642,6 +642,59 @@ getGOFPlotConfiguration <- function(plotType,
   return(plotConfiguration)
 }
 
+#' @title getBoxWhiskerPlotConfiguration
+#' @description Define a `PlotConfiguration` object
+#' @param plotScale Scale of Y Axis
+#' @param data A data.frame
+#' @param metaData dimensions and units in the data.frame
+#' @param dataMapping `DataMapping` object
+#' @param plotConfiguration A user-defined `PlotConfiguration` object
+#' @return A `PlotConfiguration` object
+#' @keywords internal
+getBoxWhiskerPlotConfiguration <- function(plotScale = "log",
+                                           colorGrouping = NULL,
+                                           data,
+                                           metaData,
+                                           dataMapping = NULL,
+                                           plotConfiguration = NULL) {
+  if (!isEmpty(plotConfiguration)) {
+    return(plotConfiguration)
+  }
```

Comment:
I see a problem wtih this implementation, because it makes it virtually impossible for users to make adjustions to the default BoxWhisker plot configuration.
But it seems to me a bigger topic (which might be also relevant for other kinds of plots), so I will create an Issue for this on GitHub, which can be adressed in further PRs